### PR TITLE
Update to ctapipe 0.10.4, ctapipe_io_lst 0.7.0

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

**Review by rlopezcoto — APPROVED** [review]
(just for the record):
main changes with respect to previous versions: removal of the option to calculate timestamp from TIB (https://github.com/cta-observatory/ctapipe_io_lst/pull/72) in ctapipe_io_lst `v0.7.0`

File: lstchain/scripts/lstchain_data_r0_to_dl1.py [hunk 1]
```diff
@@ -159,30 +140,22 @@ def main():
...
-    if args.dragon_counter0 is not None:
-        lst_event_source['EventTimeCalculator']['dragon_counter0'] = \
-            args.dragon_counter0
-    if args.ucts_t0_tib is not None:
-        lst_event_source['EventTimeCalculator']['ucts_t0_tib'] = \
-            args.ucts_t0_tib
-    if args.tib_counter0 is not None:
-        lst_event_source['EventTimeCalculator']['tib_counter0'] = \
-            args.tib_counter0
+    time_calculator = lst_event_source['EventTimeCalculator']
```
Comment (moralejo) [inline lstchain/scripts/lstchain_data_r0_to_dl1.py:143]: Won't this set the reference values to null if they are not provided in the args? (overriding possibly valid values set through the json file)

**Review by moralejo — APPROVED** [review]

**Comment by moralejo** [comment]
What is it that did not pass?

**Comment by maxnoe** [comment]
@moralejo the "required tests" have the ctapipe version in the name, so we have to change which tests we require in the settings. If we want to go ahead here, I will do this now.

**Comment by moralejo** [comment]
> @moralejo the "required tests" have the ctapipe version in the name, so we have to change which tests we require in the settings. If we want to go ahead here, I will do this now.

Ok, please merge, I'd like to try it.

**Pull request merged**